pkg/apis/performanceprofile/v2/performanceprofile_validation_test.go

```diff
@@ -217,6 +217,19 @@ var _ = Describe("PerformanceProfile", func() {
 			Expect(errors).NotTo(BeEmpty(), "should have validation error when isolated and shared CPUs have overlap")
 			Expect(errors[0].Error()).To(Or(ContainSubstring("isolated and shared cpus overlap"), ContainSubstring("shared and isolated cpus overlap")))
 		})
+		DescribeTable("should reject invalid input that does not represent CPU sets",
+			func(fieldSetter func(*PerformanceProfile, CPUSet), cpusField string) {
```

Comment:
ok, this is a backport, but still: are we guaranteed mutations made by each table item does not leak to other items?
e.g. are items fully isolated from each other?